### PR TITLE
Add example for airfield

### PR DESCRIPTION
**Pull request by swoehrl-mw**
This is the example for our new open-source tool [airfield](https://github.com/MaibornWolff/dcos-airfield), the package was introduced to the universe right before christmas.

This example references the consul example for which a separate PR is open (https://github.com/dcos/examples/pull/387)

### CODE REVIEW

File: airfield/1.11/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+* A wildcard DNS entry pointing at the loadbalancer. Each zeppelin instance will be available using a random name as a subdomain of your wildcard domain. As an example we will be using `*.zeppelin.mycorp`.
+* A Key-Value-Store to store the list of existing zeppelin instances. Currently supported are either [consul](https://www.consul.io/) or [etcd](https://coreos.com/etcd/). If you have neither installed we recommend [consul](https://github.com/MaibornWolff/dcos-consul) which is available in the [universe](https://universe.dcos.io/#/package/consul/version/latest). Check the [consul example](../../consul/1.11/README.md) in this repo for details on how to install it.
+* Enough available resources to run both Airfield and one Zeppelin instance (minimum: 3 cores, 10GB RAM).
+
+## Installation
+
+Airfield requires access to the Marathon API to manage zeppelin instances. If you are running DC/OS Enterprise you first need to create a serviceaccount for airfield:
+
+```bash
+dcos security org service-accounts keypair private-key.pem public-key.pem
```
Comment (joerg84) [inline airfield/1.11/README.md]: These are EE specific, did you test on OSS as well? Otherwise we can also make the Prereq EE...

Comment (swoehrl-mw) [reply airfield/1.11/README.md]: I tested it on OSS as well. That is why I added "If you are running DC/OS Enterprise you first need to create a serviceaccount" and provide options files for EE and OSS variants.

File: airfield/1.11/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "service_account_secret": "airfield/account-secret"
+  },
+  "airfield": {
+    "marathon_lb_base_host": ".zeppelin.mycorp",
+    "consul_endpoint": "http://api.aconsul.l4lb.thisdcos.directory:8500/v1"
+  }
+}
+```
+
+For DC/OS Open Source you need at least the following (change values to fit your cluster):
```
Comment (joerg84) [inline airfield/1.11/README.md]: Ah, could we split this into two sections OSS and EE with Oss coming first?

File: airfield/1.11/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Once you are satisfied with the configuration click on `Create new instance`. Then go back to the overview by clicking on the airfield logo on the top left side. There you can now see your newly created zeppelin instance. Wait a few minutes until the instance is running (use the refresh button on the right to check the status, the first instances will take longer to start because DC/OS has to pull the zeppelin docker image which is quite big).
+
+![Airfield Overview](img/overview02.png)
+
+Once the zeppelin instance is running click on the provided link to open it. You can now use Zeppelin however you like. As a starting point you can use the [zeppelin example](../../zeppelin/1.11/README.md) in this repo.
+
+If you no longer need your instance you can either stop it (and start it again later) or delete it from the airfield overview page.
+
+Please be aware that the zeppelin instances have no persistent storage. That means if you restart, stop or delete an instance all notebook data will be lost. So you need to export you notebooks before that.
```
Comment (joerg84) [inline airfield/1.11/README.md]: s/zeppelin/Zeppelin

**Pull request merged**